static/js/components/EducationForm.js

```diff
@@ -57,30 +53,74 @@ class EducationForm extends ProfileFormFields {
     deleteEducationEntry.call(this);
   };
 
-  renderEducationLevel(level: Option): Array<React$Element<*>|void>|void {
+  educationLevelRadioSwitch: Function = (level: Object): React$Element<*> => {
+    let radioIconStyle = {'marginRight': '8px'};
+    return (
+      <RadioButtonGroup
+        className={`profile-radio-switch ${level.value}`}
+        id={`profile-tab-education-switch-${level.value}`}
+        name={`profile-tab-education-switch-${level.value}`}
+        onChange={(event, value)=> this.handleRadioClick(value, level.value)}
+      >
+        <RadioButton value={"true"} label="Yes" iconStyle={radioIconStyle} style={{'marginRight': '30px'}} />
+        <RadioButton value={"false"} label="No" iconStyle={radioIconStyle} style={{'marginRight': '15px'}} />
+      </RadioButtonGroup>
+    );
+  };
+
+  handleRadioClick(value: string, level: string): void {
+    if (value === "true") {
+      this.openNewEducationForm(level, null);
+    }
+  }
+
+  renderEducationQuestionForm(level: Option): React$Element<*> {
+    let label = level.label,
+      prefix = label.toLowerCase().startsWith("a") ? "an" : "a",
+      levelName = !label.endsWith("degree") ? `${label.toLowerCase()} degree` : label.toLowerCase();
```

Comment:
this is a bit confusing to read, could you reformat with three separate `let` statements?